reference-data/kartothek-compat/py2-dataset/src/create_dataset.py

```diff
@@ -0,0 +1,29 @@
+import sys
+from distutils.version import LooseVersion
+from functools import partial
+
+import pandas as pd
+import storefact
+
+import kartothek
+from kartothek.io.eager.write import store_dataframes_as_dataset  # noqa
+
+assert sys.version_info.major == 2  # Show we're on Python 2
+assert LooseVersion(kartothek.__version__) < "3"  # Tested using `2.31.4`
+
+
+def get_storefactory(store_config):
+    return partial(storefact.get_store, **store_config)
+
+
+uuid = "dataset_uuid"
+store = get_storefactory({"type": "hfs", "path": "../data"})
+df = pd.DataFrame(
+    {
+        "int_col": [1, 2, 3],
```

Comment:
You should be able to achieve the same thing by using `b"int_col"` w/out going back to py2. Kartothek isn't py2 compatible anymore. This is a big coincidence that this code could even be executed.